naughty/fedora-34/2291-selinux-hostnamectl

```diff
@@ -0,0 +1 @@
+avc:  denied  { add_name } for  * comm="systemd-hostnam" name="*" scontext=system_u:system_r:systemd_hostnamed_t:s0 tcontext=system_u:object_r:init_var_run_t:s0 tclass=dir permissive=0
```

Comment:
Two things some to mind:
1. If you look at other avcs we have, it is always a good idea to stick to the pattern. So start with `* type=1400 audit(*): avc:  denied`... (Back in the day this hit us, as some pattern did not work - so might this as well, as it does not start with `*`)
2. You can test your pattern with `tests-policy`

How to use test policy:
1. Put output from test into file
2. `./tests-policy -o context < file_name` where context is for example `fedora-34` and `file_name` is the one, where you put output of the test